src/__tests__/sdk_info_test.ts

```diff
@@ -0,0 +1,69 @@
+import { ConnectWidget } from "../../src"
+import { sdkVersion } from "../../src/version"
+import { waitFor } from "../../jest/utils"
+
+const url = "https://widgets.moneydesktop.com/md/..."
+
+let container = document.createElement("div")
+let widget: ConnectWidget
+let loaded: boolean
+
+beforeAll(() => {
+  container = document.createElement("div")
+  document.body.appendChild(container)
+})
+
+afterAll(() => {
+  document.body.removeChild(container)
+})
+
+describe("Widget SDK info", () => {
+  beforeEach(() => {
+    loaded = false
+    widget = new ConnectWidget({
+      url,
+      container,
+      onLoad: () => {
+        loaded = true
+      },
+    })
+  })
+
+  afterEach(() => {
+    widget.unmount()
+  })
+
+  test("should send SDK info to the widget after the widget loads", async () => {
+    await waitFor(() => !!container?.getElementsByTagName("iframe")[0]?.src)
+
+    const iframeElement = container?.getElementsByTagName("iframe")[0]?.contentWindow
+
+    if (!iframeElement) {
+      throw new Error("Unable to find widget iframe")
+    }
+
+    const postMessageSpy = jest.spyOn(iframeElement, "postMessage")
+    const expectedPayload = JSON.stringify({
```

Comment:
I know the react-native stuff has us do `JSON.stringify` to send a post message, but should we also do that in the web versions? On the widget side, it should be able to handle both a JSON string and an object.

Reply:
I took this from your ping test 😬

Reply:
But yes, this does need to be a string value.